admin/src/main/ts/app/services/services.service.ts

```diff
@@ -105,4 +125,23 @@ export class ServicesService {
 
         return `${query}?${queryParams}`;
     }
-}
\ No newline at end of file
+
+    public isIconImage(connector: ConnectorModel): boolean {
+        return connector 
+            && connector.icon 
+            && (connector.icon.startsWith('/workspace') || connector.icon.startsWith("http"));
```

Comment:
juste pour éviter une erreur débile, je me demande si il faudrait pas faire le suivant:
```suggestion
            && (connector.icon.startsWith('/workspace') || connector.icon.startsWith("http://")) || connector.icon.startsWith("https://"));
```

Reply:
ok ouai ça sera plus sûr, je vais faire ça